src/Essentials/samples/Samples/ViewModel/FlashlightViewModel.cs

```diff
@@ -14,6 +14,7 @@ public class FlashlightViewModel : BaseViewModel
 		public FlashlightViewModel()
 		{
 			ToggleCommand = new Command(OnToggle);
+			Device.BeginInvokeOnMainThread(this.InitViewModel);
```

Comment:
drop this.

Reply:
What do you suggest instead to initialize the IsSupported property?

Reply:
Ah sorry I misunderstood, you meant remove `this.`